code-challenges.test.js

```diff
@@ -60,8 +74,85 @@ var hand2 = [5, 5, 3, 3, 4]
 var hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
 
-
-
-
+describe("when 'fullHouse' is called",() => {
+  it("returns with a boolean statement if the Tanner family (array) has one pair and three of a kind", () => {
+    expect(fullHouse(hand1)).toEqual(true)
+    expect(fullHouse(hand2)).toEqual(false)
+    expect(fullHouse(hand3)).toEqual(false)
+  })
+})
 
 // b) Create the function that makes the test pass.
+//create a function that takes an array as an arguement
+//use for loop to compare values in array
+// if array includes (i === i +1) *3 && (i === i+1)* 2
+//
+
+// const fullHouse = (tanners) => {
+//   let checkNums = tanners.every( e => {
+//     if((e === (e + 1) * 3) && (e === (e + 1) * 2)){
+//       return true
+//     }else{
+//       return false
+//     }
+//   }return checkNums
+// }
+
+//create a function that takes an array as an arguement
+//use map to iterate over array with vaule and index accessable
+//if statement that has [i] = 3 && [i] = 2 return true / else return false
+
+//filter to store 3 same values
+//filter to store 2 same values
+//these will be the count
+//find 2 unique characters and count the instances of unique characters
+//check for three of one and two of other
+
+//keep track of characters and count them -- store as variable or object
+//isolate one character and check for other instances.
+
+// const fullHouse = (tanners) => {
+//   let dupCount = []
+//   let dupValues = tanners.filter((value,i) => tanners[i] = (tanners[i + 1]))
+//     if((dupValues[0] === 2 && dupValues[1] === 3) || (dupValues[1] === 2 && dupValues[0] === 3)){
+//       return true
+//     }else {
+//       return false
+//     }
+//   }
+
+//^nope..... now to try to push into arrays.....
+
+const fullHouse = (tanners) => {
+let twoInHand = []
+let threeInHand = []
+  tanners.filter((v, i) => {
```

Comment:
The logic is there. It looks like .forEach() might be a more suitable method instead of .filter() because filter needs an arrow function that returns a boolean, whereas here some action is trying to be executed for each element in the array.